ephios/api/templates/oauth2_provider/authorized-tokens.html

```diff
@@ -0,0 +1,23 @@
+{% extends "core/settings/settings_base.html" %}
+
+{% load i18n %}
+{% block settings_content %}
+    <div class="block-center">
+    <h1>{% trans "Tokens" %}</h1>
+        <ul>
+        {% for authorized_token in authorized_tokens %}
+            <li>
+                {{ authorized_token.application }}
+                (<a href="{% url 'oauth2_provider:authorized-token-delete' authorized_token.pk %}">{% trans "revoke" %}</a>)
+            </li>
+            <ul>
+            {% for scope_name, scope_description in authorized_token.scopes.items %}
+                <li>{{ scope_name }}: {{ scope_description }}</li>
+            {% endfor %}
+            </ul>
+        {% empty %}
+            <li>{% trans "There are no authorized tokens yet." %}</li>
```

Comment:
I really don't like the look of this list or the other templates.

Reply:
Also, there should be some explanatory sentence as to what the page shows.

Reply:
I didn't want to reproduce them in the first place, but had to because of their inheritance structure. these are 1:1 copied from the library

Reply:
Want to split up the work on making the views look a little better?

Reply:
maybe @gritfessel wants to work on that after merging this PR?

Reply:
moved to #938